InteropabilityTest_Engine/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("InteropabilityTest_Engine")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("InteropabilityTest_Engine")]
+[assembly: AssemblyCopyright("Copyright ©  2020")]
```

Comment:
Copyright should be for BHoM per other toolkits

Reply:
👍 Took the liberty to update it in the other projects as well, as well as bumping their assembly file version